readme.md

```diff
@@ -119,14 +136,21 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+@artist.save - Stores an instance variable
+@artist.save!- Stores an instance variable explicitly and it cannot be changed
 
 ## Question 10
 
 Using jQuery, write an AJAX request to `http://tunr.com/artists` that would create a new artist with the name of 'Resin Laying Deer Figurine, Gold', and pop up a box saying "All done!" when complete.
 
 ```js
-// Your code starts here...
+$(document).ready(function(){
+  $.post(`http://tunr.com/artists`);
+  {
+    new artist: 'Resin Laying Deer Figurine, Gold'
+  }
+    alert("All done!")
+    });
```

Comment:
There are some syntactical errors here. I'd recommend actually trying to run this code. You'll know it's working when the only error you get is a `404` error because `http://tunr.com/artists` doesn't exist.